mopidy/zeroconf/__init__.py

```diff
@@ -0,0 +1,55 @@
+import logging
+import sys
+
```

Comment:
Add the following to make flake8 understand that Zeroconf is what this module creates and exports, and the import is thus not unused:

``` python
__all__ = ['Zeroconf']
```